gtsfm/averaging/translation/averaging_1dsfm.py

```diff
@@ -168,7 +183,7 @@ def get_prior_in_world_frame(i2, i2Ti1_prior):
             # noise_model = gtsam.noiseModel.Gaussian.Covariance(i2Ti1_prior.covariance)
             noise_model = gtsam.noiseModel.Isotropic.Sigma(3, 1e-2)
             w_i2ti1_priors.append(
-                gtsam.BinaryMeasurementPoint3(
+                BinaryMeasurementPoint3(
                     i2,
                     i1,
                     get_prior_in_world_frame(i2, i2Ti1_prior),
```

Comment:
@akshay-krishnan @ayushbaid we refer to these as i2Ti1 priors here, but I thought they were i1Ti2 priors?